src/qt/locale/bitcoin_it.ts

```diff
@@ -4310,187 +4310,187 @@ p, li { white-space: pre-wrap; }
-        <translation type="unfinished"></translation>
+        <translation>Se giá incluso nella blockchain, considera questo blocco e i blocchi antecedenti validi e ometti la verifica degli script (0 per verificare tutti i blocchi, default: %s, testnet: %s)</translation>
     </message>
     <message>
         <source>Query for peer addresses via DNS lookup, if low on addresses (default: 1 unless -connect/-noconnect)</source>
-        <translation type="unfinished"></translation>
+        <translation>Richiedi indirizzi peers tramite lookup DNS, se sono disponibili solo pochi peers (default: 1 nel caso in cui le opzioni -connect/-noconnect non siano specificate)</translation>
     </message>
     <message>
         <source>Reduce storage requirements by enabling pruning (deleting) of old blocks. This allows the pruneblockchain RPC to be called to delete specific blocks, and enables automatic pruning of old blocks if a target size in MiB is provided. This mode is incompatible with -txindex and -rescan. Warning: Reverting this setting requires re-downloading the entire blockchain. (default: 0 = disable pruning blocks, 1 = allow manual pruning via RPC, &gt;%u = automatically prune block files to stay under the specified target size in MiB)</source>
-        <translation type="unfinished"></translation>
+        <translation>Riduci lo spazio occupato su disco abilitando la cancellazione (pruning) dei blocchi più vecchi. Questo permette l&apos;utilizzazione del comando RPC pruneblockchain per cancellare blocchi specifici, e abilita la cancellazione automatica dei blocchi più vecchi se viene specificato un target massimo per lo spazio occupato su disco in MiB. Questa modalitá é incompatibile con -txindex e -rescan. Attenzione: la modifica di questa opzione richiede una nuova sincronizzazione dell&apos;intera blockchain con la rete. (default: 0 = disabilita cancellazione dei blocchi, 1 = abilita cancellazione dei blocchi via RPC, &gt;%u = cancella automaticamente i file dei blocchi per rimanere al di sotto del target massimo specificato in MiB)</translation>
     </message>
     <message>
         <source>Set lowest fee rate (in %s/kB) for transactions to be included in block creation. (default: %s)</source>
-        <translation type="unfinished"></translation>
+        <translation>Inserisci la commissione minima (in %s/kB) per l&apos;inclusione delle transazioni nei blocchi. (default: %s)</translation>
     </message>
     <message>
         <source>Sets the serialization of raw transaction or block hex returned in non-verbose mode, non-segwit(0) or segwit(1) (default: %d)</source>
-        <translation type="unfinished"></translation>
+        <translation>Imposta la serializzazione delle transazioni raw e dei blocchi esadecimali in modalitá non-verbosa, non-segwit (0) o segwit (1) (default: %d)</translation>
```

Comment:
<translation>Imposta l'ordine delle transazioni raw e dei blocchi esadecimali in modalitá non-verbosa, non-segwit (0) o segwit (1) (default: %d)</translation>

Reply:
La traduzione originale e' corretta. Serializzazione e' un termine tecnico che indica il comporre un oggetto complesso in maniera lineare  in modo che possa essere communicato o salvato in memoria. "Ordine" non sarebbe corretto in questo caso.